packages/actor-rdf-resolve-hypermedia-links-queue-wrapper-info-occupancy/README.md

```diff
@@ -0,0 +1,208 @@
+# Comunica Wrapper Info Occupancy RDF Resolve Hypermedia Links Queue Actor
+
+[![npm version](https://badge.fury.io/js/%40comunica%2Factor-rdf-resolve-hypermedia-links-queue-wrapper-info-occupancy.svg)](https://www.npmjs.com/package/@comunica/actor-rdf-resolve-hypermedia-links-queue-wrapper-info-occupancy)
+
+**Warning: Should not be used in production or in a browser environment**
+
+A comunica Wrapper Info Occupancy RDF Resolve Hypermedia Links Queue Actor.
+
+This module is part of the [Comunica framework](https://github.com/comunica/comunica),
+and should only be used by [developers that want to build their own query engine](https://comunica.dev/docs/modify/).
+
+[Click here if you just want to query with Comunica](https://comunica.dev/docs/query/).
+
+This link queue wrapper collects information about the occupancy of the underlying link queues and logs it using a [comunica loggers](https://comunica.dev/docs/query/advanced/logging/) defined by the user. It is preferable to use the [`@comunica/logger-bunyan`](https://github.com/comunica/comunica/tree/master/packages/logger-bunyan).
+
+The [`comunica-link-queue-parser-rs`](https://github.com/constraintAutomaton/comunica-link-queue-parser-rs) tie-in software can interpret the link queue event and output it as a JSON in the console and/or a file. An example output is presented below.
```

Comment:
Could you add a concrete an example usage to the readme of https://github.com/constraintAutomaton/comunica-link-queue-parser-rs ?
To show to people which commands need to be invoked in what order.

Reply:
I am not sure I understand I thought that the "installation" and "usage" section already explained it.

```bash
cargo build --release
./target/release/comunica-link-queue-parser-rs
```

and the param are all explained.

Reply:
I mean an actual concrete example, with an actual query.
The current README only shows an example output, but no example input.
The actual commands you need to execute, including comunica link traversal.

Reply:
It is done